samza-kafka/src/main/java/org/apache/samza/config/KafkaConsumerConfig.java

```diff
@@ -40,35 +42,39 @@
 
   public static final Logger LOG = LoggerFactory.getLogger(KafkaConsumerConfig.class);
 
-  static final String PRODUCER_CLIENT_ID_PREFIX = "kafka-producer";
-  static final String CONSUMER_CLIENT_ID_PREFIX = "kafka-consumer";
-  static final String ADMIN_CLIENT_ID_PREFIX = "samza-admin";
+  public static final String ZOOKEEPER_CONNECT = "zookeeper.connect";
 
+  public static final String CONSUMER_CONFIGS_CONFIG_KEY = "systems.%s.consumer.%s";
+  public static final String PRODUCER_BOOTSTRAP_SERVERS_CONFIG_KEY = "systems.%s.producer.bootstrap.servers";
+  public static final String PRODUCER_CONFIGS_CONFIG_KEY = "systems.%s.producer.%s";
```

Comment:
Should these be in KafkaConsumerConfig? Is there a KafkaConfig class/base class we can put these in?